extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md

```diff
@@ -0,0 +1,97 @@
+
+The way to specify subscriptions configurations is the same for both a classic AMP page and a Web Story. See more details about configurations in [`amp-subscriptions` configuration](https://amp.dev/documentation/components/amp-subscriptions/#configuration).
+
+The extension provides a simple UI and capabilities to surface a paywall bottomsheet, and manage blocking / unblocking locked content.
+
+> **Important Note**: Please first read API docs for [`amp-subscriptions`](https://amp.dev/documentation/components/amp-subscriptions/) and (optionally) [`amp-subscriptions-google`](https://amp.dev/documentation/components/amp-subscriptions-google/) and ensure you are set up with those basic capabilities before integrating with this component.
+
+# How it works
+
+1. Publishers can configure a Web Story with the component to implement a paywall.
+    - Paywall cannot be triggered on the first two pages of a Story or on the last page or on a Story with fewer than 4 pages.
+2. When the user navigates to the first paywalled page of the Story, <amp-story-subscriptions> will check if the user has access via <amp-subscriptions>. If yes, the user can consume the Story without any interruption.
+3. If the access is not granted, a paywall bottomsheet is shown at the bottom of the page. The paywall only prevents users from accessing the remaining pages of the Story, the user can otherwise navigate as normal (tap to previous pages, or swipe to a different Story if available).
+4. The paywall has a templated UX that allows for key messages to be customized by the publisher.
+5. Depending on publisher configuration, the dialog can support three different ways to progress:
+    - Sign-in with publisher’s sign-in flow (i.e. publisher provided sign-in page).
+    - Subscribe directly via the publisher (i.e. via a publisher provided subscription page).
+    - If enabled by the publisher, subscribe via [Subscribe with Google](https://developers.google.com/news/subscribe).
+6. If the user completes any of the above authentication flows, the user is returned to the Story page, the paywall is dismissed and the user can consume the rest of the Story.
+7. If the user encounters a subsequent paywalled Story from this publisher, they will continue to retain access based on [amp-subscription mechanisms](https://amp.dev/documentation/components/amp-subscriptions/#combining-the-amp-reader-id-with-publisher-cookies).
+
+# `amp-story` and `amp-story-subscriptions` attributes
+
+```html
+<amp-story publisher-logo-src='https://publisher-domain.com/logo.jpg'>
+    <amp-story-page>...</amp-story-page>
+    ...
+    <amp-story-subscriptions subscriptions-page-index='3' price='$0.50/week' headline='Culture at your fingertips' description='Subscribe for unlimited access.' additional-description='Cancel anytime.'>
+    </amp-story-subscription>
+</amp-story>
+```
+
+### `publisher-logo-src` {string} required
```

Comment:
Can we get it from the `amp-story` metadata? https://github.com/ampproject/amphtml/blob/main/extensions/amp-story/amp-story.md#publisher-logo-src-guidelines
Do publishers need to repeat it?

Reply:
It is from `amp-story`'s attribute. I was trying to express that this `amp-story` attribute is mandatory for this extension to work. But yes, this attribute is already mandatory. We don't need to repeatedly saying that here. Removed this entirely.